pgcli/main.py

```diff
@@ -1194,6 +1194,9 @@ def echo_via_pager(self, text, color=None):
 @click.option(
     "--warn/--no-warn", default=None, help="Warn before running a destructive query."
 )
+@click.option(
+    "--history", default=None, help="Specify history file location."
```

Comment:
Perhaps `histfile`, so it matches the name used by `psql`?

Reply:
Excellent! I knew I met `--histfile` somewhere but I forgot it's `psql`, I just haven't use psql for a long time. :)